framework/source/class/qx/event/handler/Mouse.js

```diff
@@ -375,6 +377,17 @@ qx.Class.define("qx.event.handler.Mouse",
 
       this.__fireEvent(domEvent, type, target);
 
+      /*
+       * In order to normalize middle button click events we
+       * need to fire an artificial click event if the client
+       * fires auxclick events for non primary buttons instead.
+       * 
+       * See https://github.com/qooxdoo/qooxdoo/issues/9268
+       */
+      if (type == "auxclick" && domEvent.button == 1) {
+        this.__fireEvent(domEvent, "click", target);
```

Comment:
@derrell I'm respecting our PR policy. After you wrote that the PR looks good, I thought that as a result you would approve this PR (in terms of the github PR workflow) which in my understanding is the statement of one member that the PR is ready to be merged. And of course this PR will not be merged until minimum another member approves it and the two business days grace period is over, which will be next Monday evening. Additionally I would never merge my own PR (don't know if this rule is part of our policy).